node/npm_dependencies.ts

```diff
@@ -244,12 +246,26 @@ export const vendorNPMSpecifiers = async ({
     return
   }
 
+  await fs.writeFile(
+    path.join(temporaryDirectory.path, 'polyfills.mjs'),
+    `
+    import process from "node:process";
+    import { setImmediate, clearImmediate } from "node:timers";
+    import { Buffer } from "node:buffer";
+    globalThis.process = process;
+    globalThis.Buffer = Buffer;
+    globalThis.setImmediate = setImmediate;
+    globalThis.clearImmediate = clearImmediate;
+    globalThis.global = globalThis;
+    `,
```

Comment:
I came up with this list by going through https://nodejs.org/api/globals.html, and checking which of those don't exist in Deno. I think I caught all of them.
The only ones missing are PerformanceObserver, PerformanceObserverEntryList and PerformanceResourceTiming, but something tells me that the above are way more important, and we could get away without those.